composer.json

```diff
@@ -23,19 +23,19 @@
         }
     },
     "require": {
-        "php": ">=7",
-        "amphp/parallel": "^1.1",
-        "amphp/amp": "^2.0.3",
-        "opis/closure": "^3.0.7"
+        "php": ">=7.4",
+        "amphp/parallel": "^1.4.1",
+        "amphp/amp": "^2.6.1",
+        "laravel/serializable-closure": "^1.0"
     },
     "require-dev": {
-        "amphp/phpunit-util": "^1.0",
-        "friendsofphp/php-cs-fixer": "^2.9",
-        "phpunit/phpunit": "^6.5"
+        "amphp/phpunit-util": "^2.0",
+        "friendsofphp/php-cs-fixer": "^3.5",
+        "phpunit/phpunit": "^9.5.11"
     },
     "config": {
         "platform": {
-            "php": "7.0.0"
+            "php": "7.4"
```

Comment:
I think the bump to 7.4 and migrating to laravel/serializable-closure can be done in a separate PR

Reply:
Updated 👍🏻